io.github.pieterdd.StretchBreak.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.pieterdd.StretchBreak</id>
+
+  <name>Stretch Break</name>
+  <summary>Reminds you to take regular breaks</summary>
+  <developer id="io.github.pieterdd">
+    <name>pieterdd</name>
+  </developer>
+
+  <metadata_license>MIT</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+
+  <content_rating type="oars-1.1" />
+
+  <branding>
+    <color type="primary" scheme_preference="light">#ebd1d1</color>
+    <color type="primary" scheme_preference="dark">#420925</color>
+  </branding>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>
+        https://raw.githubusercontent.com/pieterdd/StretchBreak/main/docs/mainWindow.png
```

Comment:
Screenshot URL should never reference the tip of a branch because it is no immutable.

Reply:
Replaced with permalink.